### DISCUSSION TITLE
Importing styled Blazor components in a nuget package

### DISCUSSION THREAD

**Discussion by DavidAbraham082** [tailwindlabs/tailwindcss#8950] [Help]
G'day guys!

I've got a nuget package with a bunch of Blazor components in it, with tailwind styles applied. When I import the package and reference it in another repo, the generated css doesn't include the styles referenced in the package. What's the best way to ensure they're included in the content?

I've tried creating an _Imports.razor which has `@using` directives for the imported classes, but that didn't work.
What's working now is explicitly including the required classes in my `safelist`
Is there a better way?

Thanks in advance!

**Comment by Hubinga** [comment]
G’day!

I have exactly the same problem. I also have a NuGet package with Blazor components and Tailwind, but when I use it in another project, the components are not displayed correctly because the necessary Tailwind CSS classes from the package are missing.

Have you found a solution for this yet?

Best regards, Andi

**Reply by DavidAbraham082** [reply]
Hey Andi!

This was a whole job ago so I don't still have that source code, but from memory I used prefixes in the component library and included the generated css in the output package. If you're the one maintaining the component package, this might be an option for you.

Getting the prefixes set up correctly wasn't straightforward and I had to do some manual work to add the prefixes to our existing markup.
Prefixing info [here](https://tailwindcss.com/docs/configuration#prefix)
I used a tool like [css prefix-er](https://prefixer-tailwindui.netlify.app/) to apply prefixes (I don't remember if that's the exact tool I used - just search for "tailwind prefix tool" or something similar).

**Reply by Hubinga** [reply]
Hey David!

Thanks a Lot, that fixed my problem and now everything works!
